packages/serialization/text/src/textParseNode.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
We probably shouldn't be disabling this rule for the whole file

Reply:
Done